pki.spec

```diff
@@ -645,9 +645,11 @@ Requires:         policycoreutils
 Requires:         procps-ng
 Requires:         openldap-clients
 Requires:         openssl
+Requires:         unzip
+
 Requires:         %{product_id}-tools = %{version}-%{release}
 
-Requires:         %{java_devel}
+Requires:         %{java_headless}
```

Comment:
This line probably can be removed since pki-server package already has an indirect dependency on java-headless.

Reply:
We can leave it to make the dependency on Java more explicit. I agree on moving the file outside of the jar but it can be done later.